misc/python/materialize/feature_benchmark/comparator.py

```diff
@@ -65,9 +65,14 @@ def human_readable(self) -> str:
             return f"{(1/ratio):3.1f} times faster"
 
 
-class OverlapComparator(Comparator):
+class Overlappable(Protocol):
+    def overlap(self, other: "Overlappable") -> float:
+        ...
```

Comment:
This does not seem to be a complete definition here? The idea was to call this method:

https://docs.python.org/3/library/statistics.html#statistics.NormalDist.overlap

Reply:
Oh, so `...` resolves to `Ellipsis`, so this is actually syntactic sugar ? Nevermind then.